### PR TITLE
English homework has been done

### PR DESCRIPTION
**Pull request by Alex99905**

### CODE REVIEW

**Review by Alya-Homyak — APPROVED** [review]

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+4. Does their mother cook very well?
+5. Lili is not rich, she does not drive a Mercedes.
+6. Let´s go out. It is not raining now.
+7. I have to go now. It is geting dark.
+8. A young man usually  stands at the window.
+9. Ann usually makes doll dresses herself.
+10. His wife is having coffee at the moment?
+11. Let’s go for a walk. It is not raining.
+12. Look! The boys are coming home.
+13. Maila likes Chinese food?
```
Comment (OlehHatsko) [inline homework/english/Alexandr_Konduhovich/lecture1/task1.md:17]: Does Maila like Chinese food?￼

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+14. My grandmother does not  work. She is on pension.
+15. What your brother is watching now?
```
Comment (OlehHatsko) [inline homework/english/Alexandr_Konduhovich/lecture1/task1.md:19]: What is your brother watching now￼?

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+16. Where do you live?
+17. Do you  play the piano well?
+18. Where their brother are swiming now?
```
Comment (OlehHatsko) [inline homework/english/Alexandr_Konduhovich/lecture1/task1.md:22]: Where are their brothers swimming now￼?

**Review by olgaz-skiba — APPROVED** [review]
Igfm

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+19. What they are  eating at   breakfast?
```
Comment (OlehHatsko) [inline homework/english/Alexandr_Konduhovich/lecture1/task1.md:23]: What are they eating at breakfast￼?

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+20. They are not wearing a tie today.
+22. Maila does not watches TV every morning.
```
Comment (OlehHatsko) [inline homework/english/Alexandr_Konduhovich/lecture1/task1.md:25]: Maila doesn't watch TV every morning￼

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+23. Listen! Blanca is singing in the bathroom.
+24. Bea sometimes is reads comics.
```
Comment (OlehHatsko) [inline homework/english/Alexandr_Konduhovich/lecture1/task1.md:27]: Bea sometimes reads comics￼

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md (hunk 3 above)
Comment (olgaz-skiba) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:22]: or Where is their brother swimming now?
is - brother
are - brothers

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+# Homework1 Task1
+
+### Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1. Your mum is working today?
```
[the remark this answers is not in the document]
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:5]: Тогда уже : Does your mum work today?

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+2. Why these boys are is talking now?
```
[the remark this answers is not in the document]
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:6]: Agree

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+3. Where does he live?
+4. Does their mother cook very well?
+5. Lili is not rich, she does not drive a Mercedes.
+6. Let´s go out. It is not raining now.
+7. I have to go now. It is geting dark.
+8. A young man usually  stands at the window.
+9. Ann usually makes doll dresses herself.
+10. His wife is having coffee at the moment?
```
[the remark this answers is not in the document]
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:14]: Agree

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+### Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1. Your mum is working today?
+2. Why these boys are is talking now?
+3. Where does he live?
+4. Does their mother cook very well?
+5. Lili is not rich, she does not drive a Mercedes.
+6. Let´s go out. It is not raining now.
+7. I have to go now. It is geting dark.
```
[the remark this answers is not in the document]
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:11]: Agree

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md (hunk 1 above)
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:17]: Agree

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md (hunk 2 above)
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:19]: Agree

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md (hunk 4 above)
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:23]: Agree

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md (hunk 5 above)
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:25]: Agree

File: homework/english/Alexandr_Konduhovich/lecture1/task1.md (hunk 6 above)
Comment (Alex99905) [reply homework/english/Alexandr_Konduhovich/lecture1/task1.md:27]: Agree

**Pull request merged**